### DISCUSSION TITLE
How do I filter for nonlicensed users?

### DISCUSSION THREAD

**Discussion by RobertoKGEM** [microsoft-search/pnp-modern-search#3817] [Q&A]
Is there a way to filter for unlicensed users in the refinement filter for the pnp search results webpart? The idea here is that when we onboard new employees we give them an MS license and when they leave the company we remove all licenses. I think it's a great way to filter our employee directory this way. Any advise is appreciated. Thanks!

**Comment by kasperbolarsen** [comment]
Not out of the box as the People search index does not contain any info about licensing.  However,  if you update a custom property when you assign or remove the license,  then you should be able to do it. Perhaps this resent blog post can serve as inspiration,  https://www.patrikhellgren.com/blog/customize-the-person-card-in-pnp-modern-search-people-layout/
